FernFlower-Patches/0010-LVT-Fixes-and-Support-for-Enhanced-For-loop-detectio.patch

```diff
@@ -648,9 +648,9 @@ index 0000000000000000000000000000000000000000..adc78392c4a82692b32a603e7c0aa90d
 +
 +    // remove the last break edge, if exists
 +    Statement loopcontent = loop.getFirst();
-+    if (!loopcontent.getAllSuccessorEdges().isEmpty()) {
-+      loopcontent.removeSuccessor(loopcontent.getAllSuccessorEdges().get(0));
-+    }
++//    if (!loopcontent.getAllSuccessorEdges().isEmpty()) {
++//      loopcontent.removeSuccessor(loopcontent.getAllSuccessorEdges().get(0));
++//    }
```

Comment:
Since this is ForgeFlower added code, why not just delete these lines rather than commenting them out?

Reply:
`EliminateLoopsHelper` is actually from fernflower's initial commit, [here](https://github.com/MinecraftForge/FernFlower/blob/e2d0f5d9c38561d67f23754c00addb4a3547efb2/src/de/fernflower/modules/decompiler/EliminateLoopsHelper.java)- It was commented out in the method processor with a fixme, and over time JetBrains removed the class as it was dead code. ForgeFlower simply re-adds the class. I left the code commented out as it wasn't written by ForgeFlower and it does seem to be serving a purpose, even if the end result is a bit broken. If that case with the last break edge ever comes up in Minecraft it'll be useful to have what was originally supposed to fix it there, so it can be fixed properly this time.